packages/db/src/meta/pouch/adapters/indexeddb.ts

```diff
@@ -16,21 +12,21 @@ export interface DatabasesSettings {
 }
 
 export const getDefaultSettings: GetDefaultSettings = () => ({
-  directory: path.join(Config.getTruffleDataDirectory(), ".db", "sqlite")
+  directory: path.join(Config.getTruffleDataDirectory(), ".db")
 });
 
 export class Databases<C extends Collections> extends Base.Databases<C> {
   private directory: string;
 
   setup(settings: DatabasesSettings) {
-    this.directory = settings.directory;
+    // ensure db files reside in a path that ends with indexeddb
+    // whether specified in config, or using default
+    this.directory = path.join(settings.directory, "indexeddb");
     fse.ensureDirSync(this.directory);
-
-    PouchDB.plugin(PouchDBNodeWebSQLAdapter);
   }
 
   createDatabase(resource) {
     const savePath = path.resolve(this.directory, resource);
-    return new PouchDB(savePath, { adapter: "websql" });
+    return new PouchDB(savePath); // uses IndexedDB
```

Comment:
It uses `indexeddb` by default? Should we specify it here for clarity? Maybe the comment is enough...

Reply:
I think the comment conveys that, but am happy to change it to be more specific is you wish.